datalake/src/test/java/com/sequenceiq/datalake/flow/SdxReactorFlowManagerTest.java

```diff
@@ -70,10 +74,14 @@ class SdxReactorFlowManagerTest {
 
     private SdxCluster sdxCluster;
 
+    @Mock
+    private Promise<AcceptResult> acceptResultPromise;
+
     @BeforeEach
-    void setUp() {
+    void setUp() throws InterruptedException {
         sdxCluster = getValidSdxCluster();
-        BaseFlowEvent baseFlowEvent = new BaseFlowEvent("dontcare", 1L, "crn");
+        when(acceptResultPromise.await(anyLong(), any())).thenReturn(FlowAcceptResult.runningInFlow("flowId"));
```

Comment:
This test class took 1 minute to execute. The reason is because the promised timed out after 10 seconds and the workaround was the try-catch in individual tests. I removed them and created a mock promise that returns immediately. Now it takes ~1sec to execute all tests.